sdk/storage/storage-file/ChangeLog.md

```diff
@@ -1,18 +1,44 @@
 # Changelog
 
+2019.07 Version 11.0.0-preview.1
+
+- [Breaking] Client types are renamed from *URL to *Client.
+  - ServiceURL, ShareURL, DirectoryURL and FileURL to ServiceClient to FileServiceClient, ShareClient, DirectoryClient and FileClient respectively.
+- [Breaking] Aborter parameters are now moved into option bags.
+  - `abortSignal` attrubute(optional) in the option-bag of respective module has to be utitlized for the `Aborter.timeout(<milliseconds>)` functionality.
+  - `Aborter.none` is the default value.
+- [Breaking] I- prefixes are removed from interface names.
+  - Example- `IDirectoryCreateOptions` is updated to `DirectoryCreateOptions`, the new names have to be used.
+- [Breaking] The static methods to create client types are removed. The functionality is moved into new instance methods added to the parent clients.
+- [Breaking] The telemetry strings have been updated.
+  - `Azure-Storage/${SDK_VERSION}` is updated to `azsdk-js-storageblob/${SDK_VERSION}`.
+- [Breaking] `withPipeline()` method is removed.
+- Async iterators with pagination support are added for listing methods
+  - `listFilesAndDirectories()` and
+  - `listShares()`
+- [Breaking] Methods that list segments(`listFilesAndDirectoriesSegment()` and `listSharesSegment()`) are no longer exposed in public api.
+- [Breaking] High level convenience functions are moved into clients as their instance member function.
+- [Breaking] `StorageClient` is no longer exposed. `StorageClient.newPipeline()` static method is moved to the top level exported function `newPipeline()`.
+- Constructor overloads added into client types so they can be constructed from a url and a pipeline/credential and connection string.
+  - Connection string method is supported only in Node.js (not browsers).
+- Creation/Deletion of child resources are duplicated to parent client type.
+- HTTP proxy support is added (Node.js only)
```

Comment:
link to the sample showing the usage?

Reply:
You can add a simple sample code here, or the sample file path. But a link maybe outdated in the future.